src/components/Chat/MessageList/Message/index.js

```diff
@@ -23,14 +23,18 @@ const Message = props => {
     pinPost,
   } = props
 
+  const [messageText, setMessageText] = useState(text)
   // Adds the text to be used for the date divider
   const today = new Date().toLocaleDateString()
   const dateText = dateSent === today ? 'Tänään' : dateSent
 
   // Checks if message is combined user activity message
-  const isSystemCombinedUserActivity = () =>
-    type === 'system_combined_user_activity'
-
+  const isSystemCombinedUserActivity = () => {
```

Comment:
Tiny nit: I think the naming here is a little misleading since `isSystemCombinedUserActivity` function is now returning true for other message types than 'system_combined_user_activity'. Maybe just rename the function? But it works great! :)

Reply:
Changed this to better describe what's going on :)